README.md

```diff
@@ -26,35 +28,62 @@ This project provides an easy to use developers environment for the **RIF Market
 # Dependencies
 
 ## Prerequisities
-0. node v10 (or [nvm](https://github.com/nvm-sh/nvm) with node v10 installed)
-1. [Docker](https://www.docker.com/)
-2. [Docker compose](https://docs.docker.com/compose/install/)
-3. [IPFS](https://ipfs.io/) 
-The recommended way to install IPFS is using [ipfs-update](https://github.com/ipfs/ipfs-update) but there are other ways described [here](https://github.com/ipfs/go-ipfs#install) as well. Recommended is to run the latest version but required is at least `0.7.0`.
-
+0. node v10 (or [nvm](https://github.com/nvm-sh/nvm) or [n](https://github.com/tj/n) with node v10 installed (**to be upgraded**))
```

Comment:
does it still not work with node v12?
node v10 is deprecated now: https://nodejs.org/en/about/releases/

```suggestion
0. node v10 (installed directly, or using your favourite Node.js manager, such as [fnm](https://github.com/Schniz/fnm), [nvm](https://github.com/nvm-sh/nvm), [n](https://github.com/tj/n), etc)(**to be upgraded**)
```

Reply:
yep